exonum-light-client/README.md

```diff
@@ -5,9 +5,9 @@ Java client for [Exonum blockchain][exonum].
 Exonum light client is Java library for working with Exonum blockchain
 from the client side and can be easily integrated to an existing 
 Java application.  
-Also, Exonum light client provides access to [common utils][ejb-common]
-toolkit which contains some helpful functions for _hashing_,
-_cryptography_, _serialization_ etc. 
+Also, Exonum light client provides access to [common utils][ejb-documentation]
```

Comment:
direct link to common package?

Reply:
I tried to find it, there is no such page, unfortunately. As you suggested, we may try adding `package-info.java` to see if it will help.